internal/gengapic/testdata/empty_client_init.want

```diff
@@ -41,37 +58,40 @@ func NewClient(ctx context.Context, opts ...option.ClientOption) (*Client, error
 	if err != nil {
 		return nil, err
 	}
-	c := &Client{
+	client := Client{CallOptions: defaultCallOptions()}
+
+	c := &GrpcClient{
 		connPool:    connPool,
 		disableDeadlines: disableDeadlines,
-		CallOptions: defaultCallOptions(),
-
 		client: mypackagepb.NewFooClient(connPool),
+		CallOptions: &client.CallOptions,
 	}
 	c.setGoogleClientInfo()
 
-	return c, nil
+	client.internalClient = c
+
+	return client, nil
 }
 
 // Connection returns a connection to the API service.
 //
 // Deprecated.
-func (c *Client) Connection() *grpc.ClientConn {
```

Comment:
This appears to be a breaking change.

Reply:
It's handled by the embedded interface; this is lost in the golden tests.

Reply:
The interface embedding only brings the methods of the `internalClient` interface, not all of the methods of the `GRPCClient` type. See https://play.golang.org/p/WGSV7m8VDWN.

Reply:
Connection has now been added